llama_stack/distribution/build_venv.sh

```diff
@@ -49,61 +47,117 @@ ENVNAME=""
 SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
 source "$SCRIPT_DIR/common.sh"
 
+# check if a command is present
+is_present() {
+  command -v "$1" &>/dev/null
+}
+
+# pre-run checks to make sure we can proceed with the installation
+pre_run_checks() {
+  local env_name="$1"
+
+  # checking if uv is installed
```

Comment:
nit: this comment is not necessary. in general, I find that the less "tokens" there are in code without it being unclear, the better it is.